app/screens/settings/notification_settings_email/index.js

```diff
@@ -1,19 +1,47 @@
 // Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
+import {bindActionCreators} from 'redux';
```

Comment:
by doing what I mentioned above with platform specific files you can avoid having this file duplicate